v3/src/components/case-tile-common/attribute-format-utils.tsx

```diff
@@ -97,8 +97,10 @@ export function renderAttributeValue(str = "", num = NaN, attr?: IAttribute, key
 }
 
 export const findLongestContentWidth = (attr: IAttribute) => {
-  // include attribute name in content width calculation
-  let longestWidth = Math.max(kMinAutoColumnWidth, measureText(attr.name, kCaseTableHeaderFont))
+  // We do not include attribute name in content width calculation because header text can be split over
+  // two lines and ellided
+  const headerWidth = measureText(attr.name, kCaseTableHeaderFont)
+  let longestWidth = Math.max(kMinAutoColumnWidth, Math.ceil(3 + (headerWidth/2)))
```

Comment:
I'm a bit conflicted here. I think it does make sense to include the column header in the calculation, but I agree with the critique that my previous approach was too simplistic. With the previous minimum width of 40, using the `sample=mammals` data, auto-fit resulted in `LifeSpa⏎n`, which seems wrong. Your change to include `headerWidth/2` in the computation (since the header can be split across two lines) doesn't help for that case. What I think I want as a user is for the first word of the attribute name not to be broken across multiple lines _within reason_. So maybe that means considering half of the entire width of the label (including units -- which are not currently factored in) _and_ considering the width of the first word of the attribute name, but limiting their effect -- like maybe the header width has a maximum contribution of 150, while the maximum data width is 600, or something like that.

On the other hand, I understand the desire to not spend any more time on this, in which case a compromise might be to raise the minimum width to 50 (which fixes the `LifeSpa⏎n` issue) and leaving the rest of the computation you have in place. 🤷